### PR TITLE
Update to latest clicumber in go mod

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: test/integration/crcsuite/crcsuite.go [hunk 1]
```diff
@@ -137,9 +137,13 @@ func FeatureContext(s *godog.Suite) {
 
 	s.BeforeScenario(func(pickle *messages.Pickle) {
 		// copy data/config files to test dir
-		err := CopyFilesToTestDir()
-		if err != nil {
-			os.Exit(1)
+		for _, tag := range pickle.GetTags() {
```
Comment (cfergeau) [inline test/integration/crcsuite/crcsuite.go:141]: Is this a 'valid' use of godog tags? (change test behaviour depending on the tag, as opposed as only a way of filtering which tests are run)

File: test/integration/crcsuite/crcsuite.go [hunk 2]
```diff
@@ -80,10 +80,11 @@ func FeatureContext(s *godog.Suite) {
 			usr, _ := user.Current()
 			CRCExecutable = filepath.Join(usr.HomeDir, "go", "bin")
 		}
+		OCExecutable := filepath.Join(CRCHome, "bin", "oc")
```
Comment (cfergeau) [inline test/integration/crcsuite/crcsuite.go:83]: This does not need to be exported, so `ocExecutable` should work too. However this commit makes me wonder if the clicumber changes were not too drastic, as now a clicumber test case can no longer change environment variables, which was the case before. Wondering if there could be other side effects.

File: test/integration/features/story_marketplace.feature [hunk 3]
```diff
@@ -41,7 +41,6 @@ Feature: Operator from marketplace
     Scenario: Failover
         # simulate failure of 1 pod, check that it was replaced
         When executing "POD=$(oc get pod -o jsonpath="{.items[0].metadata.name}")" succeeds
-        And executing "echo $POD" succeeds
         And executing "oc delete pod $POD --now" succeeds
```
Comment (cfergeau) [inline test/integration/features/story_marketplace.feature:44]: Is 
```
        When executing "POD=$(oc get pod -o jsonpath="{.items[0].metadata.name}")" succeeds
        And executing "oc delete pod $POD --now" succeeds
```
still working as expected with the new clicumber ?

**Comment by praveenkumar** [comment]
/close 

Current clicumber update doesn't work as user expected behavior, which again need more update and testing.

**Pull request closed without merging**